### ISSUE TITLE
Term for mandates to wear PPE/masks

### ISSUE THREAD

**Issue by sebastianduesing** [infectious-disease-ontology-extensions/ido-covid-19#2]
I'm in process of importing terms from APOLLO-SV to populate the "plan specification" class. While looking over APOLLO-SV's "infectious disease control strategy" class, I noticed that there's no class for mask mandates/other control strategies that involve PPE, and that feels like an important term for IDO-COVID-19 to have. I looked around to see if a term like that exists in another ontology but couldn't find one. Is there already a class for that? If not, should I add one?

**Comment by PhiBabs935** [comment]
Not a direct answer to your question, but something else we should think about in connection to this issue: whether these terms should also go into VIDO, and IDO Core as well. In particular, I could see mask mandate control strategies being relevant to viral pandemics in general, so at the very least the relevant terms may go into VIDO as well.

A while back, there was actually an agreement to add terms for social distancing, hand washing and face mask wearing to IDO core and John had said he would update (not sure whether he did so). See here: https://github.com/infectious-disease-ontology/infectious-disease-ontology/issues/6

**Comment by sebastianduesing** [comment]
That's a fair call. I think there's definitely a case to be made for terms for mask mandate control strategies to go into VIDO, and it seems plausible that "personal protective equipment" and similar terms may be best placed in IDO Core. I checked out IDO Core, and (at least in the version available on Bioportal), it seems like those terms aren't in there yet.

**Comment by PhiBabs935** [comment]
Yeah, the version on Bioportal hasn't been updated since 2017. Lindsay Cowell, who is in charge of IDO, still has to vet John's changes at some point.

**Comment by sebastianduesing** [comment]
Got it.

**Comment by johnbeve** [comment]
FYI I updated the IDO file last night with several terms, etc. Barry gave us the green light to go ahead; I'll keep updating as we make progress.
